torch/__init__.py

```diff
@@ -368,10 +378,16 @@ def _load_global_deps() -> None:
         ]
         if not is_cuda_lib_err:
             raise err
+
         for lib_folder, lib_name in cuda_libs.items():
             _preload_cuda_deps(lib_folder, lib_name)
+
         ctypes.CDLL(global_deps_lib_path, mode=ctypes.RTLD_GLOBAL)
 
+    except Exception:
```

Comment:
Ugh... this is dangerous. Can we at least log the exception or something?! Shouldn't this be SystemError, ImportError or something more specific?

Reply:
I mean shouldn't we want to hard file sometimes?